proposals/NNNN-rc-sdl-module.md

```diff
@@ -0,0 +1,239 @@
+# Remote Control - SDL Module
+
+* Proposal: [SDL-nnnn](NNNN-rc-sdl-module.md)
+* Author: [Joey Grover](https://github.com/joeygrover/)
+* Status: **Awaiting Review**
+* Impacted Platforms: [RPC]
+
+## Introduction
+
+There is a strong desire to implement some method of inter-app communication between SDL apps. The SDL protocol can be enhanced to facilitate this feature. Leveraging the previously accepted Remote Control functionality, we can introduce a new module type, SDL_APP.
+
+
+## Motivation
+
+SDL connected apps provide augmented functionality to an SDL enabled IVI system. This includes mobile navigation or even off-board voice recognition. Most apps focus on a particular feature or experience they provide and therefore leverage libraries, 3rd party solutions, or other apps to perform the features that not part of their main line user experience. One of the ways an app augments itself is using SDL enabled features, send location, voice req, etc.
```

Comment:
In the fourth sentence, please update to "...the features that are not part of their mainline user experience."

Additionally, I think this section should include reference to SDL 0130 if it is meant to serve as an alternative to that proposal.